test/jdk/jdk/crac/java/net/InetAddress/ResolveTest.java

```diff
@@ -96,11 +98,16 @@ private static Future<?> startTestProcess() throws Exception {
         cmd.addAll(Arrays.asList("--name", CONTAINER_NAME));
         cmd.add(imageName);
         // Checkpointing does not work for PID 1, therefore we add an intermediary bash process
+        // At the same time docker --init is not sufficient; since `tini` does not wait for *all*
+        // children to finish, just the main process, creating /cr/cppath might race and we would
+        // not be able to restore the process.
+        // This workaround should not be necessary when https://github.com/openjdk/crac/pull/46
+        // is integrated.
         List<String> javaCmd = new ArrayList<>();
-        javaCmd.addAll(Arrays.asList("/jdk/bin/java", "-cp /test-classes/", "-XX:CRaCCheckpointTo=/cr"));
+        javaCmd.addAll(Arrays.asList("/jdk/bin/java", "-cp", "/test-classes/", "-XX:CRaCCheckpointTo=/cr"));
```

Comment:
This looks belonging to #50. Could this be the reason for strange failures of this test I've reported there?

Reply:
Why would you think that? There's a different variant of this test in #50. This particular change, while correct, has no effect when we concatenate the list anyway, it's there only for clarity. I had to change the line when experimenting with `--init` version without the bash loop workaround.